polars/polars-core/src/chunked_array/ops/aggregate.rs

```diff
@@ -804,6 +815,21 @@ impl QuantileAggSeries for Utf8Chunked {
     }
 }
 
+#[cfg(feature = "dtype-binary")]
+impl QuantileAggSeries for BinaryChunked {
```

Comment:
These can be removed. The `Series` can just return `null` for the Binary dtype

Reply:
I get this compilation error when I remove this:
```
error[E0277]: the trait bound `datatypes::BinaryType: datatypes::PolarsIntegerType` is not satisfied
   --> polars/polars-core/src/series/implementations/binary.rs:326:45
    |
326 |         QuantileAggSeries::median_as_series(&self.0)
    |         ----------------------------------- ^^^^^^^ the trait `datatypes::PolarsIntegerType` is not implemented for `datatypes::BinaryType`
    |         |
    |         required by a bound introduced by this call
    |
    = help: the following other types implement trait `datatypes::PolarsIntegerType`:
              datatypes::Int16Type
              datatypes::Int32Type
              datatypes::Int64Type
              datatypes::Int8Type
              datatypes::UInt16Type
              datatypes::UInt32Type
              datatypes::UInt64Type
              datatypes::UInt8Type
note: required for `chunked_array::ChunkedArray<datatypes::BinaryType>` to implement `chunked_array::ops::aggregate::QuantileAggSeries`
   --> polars/polars-core/src/chunked_array/ops/aggregate.rs:715:9
    |
715 | impl<T> QuantileAggSeries for ChunkedArray<T>
    |         ^^^^^^^^^^^^^^^^^     ^^^^^^^^^^^^^^^

error[E0277]: the trait bound `datatypes::BinaryType: datatypes::PolarsIntegerType` is not satisfied
   --> polars/polars-core/src/series/implementations/binary.rs:339:47
    |
339 |         QuantileAggSeries::quantile_as_series(&self.0, quantile, interpol)
    |         ------------------------------------- ^^^^^^^ the trait `datatypes::PolarsIntegerType` is not implemented for `datatypes::BinaryType`
    |         |
    |         required by a bound introduced by this call
    |
    = help: the following other types implement trait `datatypes::PolarsIntegerType`:
              datatypes::Int16Type
              datatypes::Int32Type
              datatypes::Int64Type
              datatypes::Int8Type
              datatypes::UInt16Type
              datatypes::UInt32Type
              datatypes::UInt64Type
              datatypes::UInt8Type
note: required for `chunked_array::ChunkedArray<datatypes::BinaryType>` to implement `chunked_array::ops::aggregate::QuantileAggSeries`
   --> polars/polars-core/src/chunked_array/ops/aggregate.rs:715:9
    |
715 | impl<T> QuantileAggSeries for ChunkedArray<T>
    |         ^^^^^^^^^^^^^^^^^     ^^^^^^^^^^^^^^^


```